src/main/java/com/jalasoft/webservice/model/OCRExtractor.java

```diff
@@ -1,44 +1,48 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
 package com.jalasoft.webservice.model;
 
 import java.io.File;
-
-
+import java.io.IOException;
 import net.sourceforge.tess4j.ITesseract;
 import net.sourceforge.tess4j.TesseractException;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
-
 import net.sourceforge.tess4j.Tesseract;
 
 /**
- * Copyright (c) 2019 Jalasoft.
+ * The class implements methods to convert PDF files into image files, Creates the Controller Model using Tesseract
+ * with the wrapper Tess4J.
  *
- * This software is the confidential and proprietary information of Jalasoft.
- * ("Confidential Information"). You shall not
- * disclose such Confidential Information and shall use it only in
- * accordance with the terms of the license agreement you entered into
- * with Jalasoft.
+ * @author Andy Bazualdo on 9/23/19.
+ * @version 1.0
  */
-public class OCRExtractor {
+public class OCRExtractor implements IConvert {
+    private OCRCriteria ocrCriteria;
+
     /**
-     * Creates the Controller Model using Tesseract with the wrapper Tess4J.
      *
-     * @param file for use it as file to extract the text.
-     * @param lang language of the file and for using the data training.
-     * @return String with the text of the image.
-     * @throws To implement after this demo
+     * @param criteria the object have the parameters to start the method execution
+     * @return returns the result of the method and error message in case of error
+     * @throws IOException get input/output exception to read files
      */
-    public String extract(String filePath, String lang){
-        File imageFile = new File(filePath);
+    @Override
+    public String convert(Criteria criteria) throws IOException {
+        ocrCriteria = (OCRCriteria) criteria;
+        File imageFile = new File(ocrCriteria.getFilePath());
         ITesseract tesseract = new Tesseract();
         tesseract.setDatapath("../../../../ThirdParty/Tess4J/tessdata/"); // path to tessdata directory
-        tesseract.setLanguage(lang);
+        tesseract.setLanguage(ocrCriteria.getLang());
         try {
             String result = tesseract.doOCR(imageFile);
             return result;
         } catch (TesseractException e){
-            return "e.getMessage()";
+            return e.toString();
```

Comment:
Review this line because I think that this line not displays the message of the error.